api.md

```diff
@@ -469,7 +531,77 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
 
+This function is typically used if a data node should no longer be
```

Comment:
Is this paragraph needed?  If so "need" >> "needs"

Reply:
yeah, I think it's fine to reinforce the concept. Updated the error.